bokeh/io/notebook.py

```diff
@@ -454,17 +457,23 @@ def show_app(app, state, notebook_url, port=0):
 
     loop = IOLoop.current()
 
+    if not proxy_url_func:
+        proxy_url_func = _server_url
+
     origin = _origin_url(notebook_url)
     server = Server({"/": app}, io_loop=loop, port=port,  allow_websocket_origin=[origin])
 
     server_id = uuid4().hex
     curstate().uuid_to_server[server_id] = server
 
     server.start()
-    url = _server_url(notebook_url, server.port)
+    url = proxy_url_func(notebook_url, server.port)
+
+    logging.debug("Server URL is %s" % url)
+    logging.debug("Origin URL is %s" % origin)
 
     from ..embed import server_document
-    script = server_document(url)
+    script = server_document(url, resources=None)
```

Comment:
Why are you passing `resources=None` here? AFAIK this will cause normal usage to break.

Reply:
I thought normal bokeh server code doesn't go through this pathway - only the notebook loading.  For our case, and I thought was the general case, output_notebook is always assumed to be called and has the javascript.  This seemed to try to load the javascript on each cell, which seemed excessive.  Also, the URLs there were munged in a way that had them 404.  I could do more work on server_document to fix more of the URL parsing, but was also afraid of breaking things.  If memory serves me correctly, it assumes again that all the JS being served is right off the host, and doesn't have any additional path items from the URL.

Reply:
Hrmm, I guess I will have to test it out. I had not considered that the resources loaded by `output_notebook` would be accessible to a notebook app, but perhaps that is in fact the case.